class.jetpack-cli.php

```diff
@@ -304,8 +305,43 @@ public function reset( $args, $assoc_args ) {
 				WP_CLI::success( __( 'Modules reset to default.', 'jetpack' ) );
 				break;
 			case 'prompt':
-				WP_CLI::error( __( 'Please specify if you would like to reset your options, or modules', 'jetpack' ) );
+				WP_CLI::error( __( 'Please specify if you would like to reset your options, modules or sync-checksum', 'jetpack' ) );
 				break;
+			case 'sync-checksum':
+				global $wpdb;
+				$option = 'jetpack_callables_sync_checksum';
+				if ( is_multisite() && function_exists( 'get_sites' ) ) {
+					$sites = get_sites( array( 'number' => 1000 ) );
+					$count_fixes = 0;
+					foreach ( $sites as $site ) {
+						switch_to_blog( $site->blog_id );
+						$count = (int) $wpdb->get_var( $wpdb->prepare( "SELECT COUNT(*) FROM $wpdb->options WHERE option_name = %s", $option ) );
+						if ( $count > 1 ) {
+							delete_option( $option );
+							WP_CLI::line( sprintf( __( 'Deleted %s %s options from %s', 'jetpack' ), $count, $option, "{$site->domain}{$site->path}" ) );
+							$count_fixes++;
+							sleep( 20 ); // Allow some time for replication to catch up.
```

Comment:
20 seconds is a lot - if there are several sites with multiple checksum options, don't we risk to hit the max execution time that way?

Reply:
I am not sure what `wp-cli` command maximum execution time is. But you make a very good point. 
It seems that some servers are set up to only have an maximum execution time of 300 sec.
Which would mean that we could delete the option for about 15 sites at a time. Which is not ideal.